workflows/config/snakemake/production-config.yml

```diff
@@ -1,9 +1,9 @@
 "seed": 12345
 "population_id": 0
-"num_samples_per_population": [10, 10, 10]
-"replicates": 20
+"replicates": 3
```

Comment:
should this be changed? it looks like you probably want to create a new `config` file for the no annotation runs